README.md

```diff
@@ -6,20 +6,32 @@
 
 ADaM in R Asset Library
 
-### Purpose
+## Purpose
 
 To provide an open source, modularized toolbox that enables the pharmaceutical programming community to develop ADaM datasets in R.
 
+## Installation
 
-### Scope
+Once the package is available from CRAN you'll be able to install it using `install.packages("admiral")`.
+
+In the meantime you can install the latest release of the package directly from GitHub.
+
+```r
+if (!requireNamespace("remotes", quietly = TRUE)) {
+  install.packages("remotes")
+}
+remotes::install_github("Roche-GSK/admiral")
+```
+
+## Scope
 
 * Build a toolbox of re-usable functions and utilities to create ADaM datasets in R in a modular manner
```

Comment:
very minor comment maybe - instead of "...in R in a modular manner." can we write "... using R scripts in a modular manner".